src/api/v1beta1/oneagent_types.go

```diff
@@ -29,6 +29,11 @@ type OneAgentSpec struct {
 }
 
 type CloudNativeFullStackSpec struct {
+	// Optional: the Dynatrace installer container image
+	// Defaults to the registry on the tenant for both Kubernetes and for OpenShift
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="CodeModulesImage",order=12,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced","urn:alm:descriptor:com.tectonic.ui:text"}
+	CodeModulesImage string `json:"codeModulesImage,omitempty"`
```

Comment:
couldn't this be in the `AppInjectionSpec` ?

Reply:
so you basically mean to remove the field out of the ApplicationMonitoringSpec and the CloudNativeFullStackSpec and just add it to the AppInjectionSpec?

Reply:
yas :)